draft-ietf-quic-transport.md

```diff
@@ -7334,6 +7334,38 @@ change controller of the IETF and a contact of the QUIC working group
 (quic@ietf.org).
 
 
+## QUIC Versions Registry {#iana-version}
+
+IANA \[SHALL add/has added] a registry for "QUIC Versions" under a "QUIC"
+heading.
+
+The "QUIC Versions" registry governs a 32-bit space; see {{versions}}. This
+registry follows the registration policy from {{iana-policy}}. Permanent
+registrations in this registry are assigned using the Specification Required
+policy ({{!RFC8126}}).
+
+All fields in {{iana-provisional}} MUST be included in all registrations for
+this registry.
```

Comment:
I'm not sure how to square away this MUST with the MAY exception in {{iana-provisional}}: "Provisional registrations MAY omit the Specification and Notes fields, plus any additional fields that might be required for a permanent registration." Presumably the MAY is still allowed ... why is this MUST necessary here?